docs/api/segmentrules.md

```diff
@@ -0,0 +1,64 @@
+# Segment Rules
+
+Adds ability to create, list, update, delete and retrieve Segment Rules.
+
+Once a Segment Rule is created it's stored in the database and can be retrieved via the API described below.
+
+## API Endpoints
+
+### POST /api/v1/segment-rules
+
+Creates a new segment rule, requires a `name`, `active` `target_content_type`, `target_field` and `target_field_value`. The `target_content_type` can be any of the allowed content types, `target_field` is a valid field of defined `target_content_type`, `active` has either true or false and `target_field_value` is desired value for `target_field`.
```

Comment:
Would be nice to have these each on its own line, for clarity, and identify what kind of data is expected to be input